bv_mission:
  ros__parameters:
    mission_waypoints: [
      47.39792614, 8.54609739, 15.24,
      47.3981150, 8.5464308, 15.24,
      47.3982126, 8.5460077, 15.24,
      47.3980250, 8.5458851, 15.24
    ]
    max_velocity: 5.0
    time_to_max:   1.0
    takeoff_alt:  15.24
    waypoint_tolerance: 1.0 # meters